platform/config-core/src/main/java/org/motechproject/config/core/utils/ConfigPropertiesUtils.java

```diff
@@ -115,6 +117,39 @@ public static Properties getPropertiesFromSystemVarString(String string) {
     }
 
     /**
+     * Returns {@code PropertiesConfiguration} and creates file for this configuration if it does not exist
+     * @param basePath path to the file with config
+     * @param fileName name of the file with config
+     */
+    public static PropertiesConfiguration getPropertiesConfiguration(String basePath, String fileName) {
+        createFileIfDoesNotExist(basePath, fileName);
+
+        PropertiesConfiguration propertiesConfiguration = new PropertiesConfiguration();
+        propertiesConfiguration.setBasePath(basePath);
+        propertiesConfiguration.setFileName(fileName);
+
+        try {
+            propertiesConfiguration.load();
+        } catch (ConfigurationException e) {
+            throw new MotechConfigurationException(String.format("Cannot load configuration from: %s",
+                    propertiesConfiguration.getPath()), e);
+        }
+
+        return propertiesConfiguration;
+    }
+
+    private static void createFileIfDoesNotExist(String basePath, String fileName) {
+        File configFile = new File(basePath, fileName);
+
+        try {
+            new File(configFile.getParent()).mkdirs();
```

Comment:
please add this brief comment that this methods creates file only if a file with this name does not yet exist